### PR TITLE
Feature: add geetest captcha

### PR DESCRIPTION
**Pull request by oyershov**

### CODE REVIEW

File: src/containers/GeetestCaptcha/index.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    private captchaContainerRef;
+    private captcha;
+
+    public componentDidMount() {
+        this.props.geetestCaptchaFetch();
+    }
+
+    public componentWillReceiveProps(next: Props) {
+        if (this.props.geetestCaptchaKeys !== next.geetestCaptchaKeys && next.geetestCaptchaKeys !== undefined) {
```
Comment (josadcha) [inline src/containers/GeetestCaptcha/index.tsx]: why do you check if `next.geetestCaptchaKeys !== undefined` if you can just check if `&& next.geetestCaptchaKeys`

**Comment by pkucherenk0** [comment]
The PR seems good, but issue with back end was found that is blocking this Pr from being approved.

**Comment by pkucherenk0** [comment]
QA Approve

File: src/components/SignUp/SignUp.test.tsx [hunk 2]
```diff
@@ -59,7 +60,7 @@ describe('SignUp component', () => {
 
     it('should render captcha block', () => {
         const wrapper = setup({hasConfirmed: true, captchaType: 'recaptcha'});
-        expect(wrapper.find('.cr-sign-up-form__recaptcha').exists()).toBe(true);
+        expect(wrapper.find('.cr-sign-up-form__recaptcha').exists()).toBe(false);
```
Comment (josadcha) [inline src/components/SignUp/SignUp.test.tsx]: if type is 'recaptcha' block '.cr-sign-up-form__recaptcha' are checking to not exist?

Comment (oyershov) [reply src/components/SignUp/SignUp.test.tsx]: because we have ```captchaType: 'none',```

**Pull request merged**